lib/config/defaultTsConfig.ts

```diff
@@ -0,0 +1,413 @@
+import httpRoutes from './httpRoutes'
```

Comment:
Why keeping both the `default.config.js` and this file? You should only have one file for the default configuration and it should be in TS